downloads/swan-lake-release-notes/swan-lake-preview8/RELEASE_NOTE.md

```diff
@@ -0,0 +1,519 @@
+                        
+> `ballerina update`
+
+ Next, execute the command below to update to Swan Lake Preview 8.
+
+ > `ballerina dist pull slp8`                 
+
+However, if you are using a Ballerina version below 1.1.0, install via the [installers](https://ballerina.io/downloads/).
+
+##### For New Users
+
+If you have not installed Ballerina, then download the [installers](https://ballerina.io/downloads/) to install.
+
+#### Highlights
+
+- Ability to use `\` to escape the special characters in an identifier without mentioning the initial quote
+- Ability to specify included record parameters
+- Introduction of service typing changes basing services on objects
+- Extension of Ballerina transaction capabilities to define transactional resource functions and transactional remote functions
+- Improvements to the HTTP, Log, Email, WebSub, and UUID Standard Library modules
+- Introduction of the new GraphQL, NATS Streaming (STAN), and WebSocket Standard Library modules
+
+### What is new in Ballerina Swan Lake Preview 8
+
+#### Language
+
+##### Support Identifier Escapes Without an Initial Quote
+
+Now, `\` can simply be used to escape the special characters in an identifier without mentioning the initial quote. For example, both the formats below are supported now.
+
+```ballerina
+int a\-b;
+int 'a\-b;
+```
+
+##### Included Record Parameters
+
+Included record parameters can be specified as `*T P` in which `T` denotes the record type descriptor and `P` denotes the name of the parameter. When this is compared with the required parameter, the difference is that the values, which are used for the fields of the included record parameter behave as named arguments for the function call, if it had been declared as parameters of the function.
+
+The field names of all included record parameters, which are not of the `never` type, must be distinct from each other and also from the names of the parameters.
+
+```ballerina
+type Student record {
+	string firstName;
+	string secondName;
+	int age;
+	Address address?;
+	never gender?;
+};
+
+type Address record {|
+	string city;
+	string country;
+|};
+
+type Grades record {|
+	never maths?;
+	never physics?;
+	never chemistry?;
+
+	int...;
+|};
+
+function studentDetails(int addmissionNo, string addmissionDate, *Student student) {
+	string fullName = <string>student.firstName  + <string>student.secondName;
+	int age = student.age;
+	if (student["address"] is Address) {
+    	Address address = <Address>student["address"];
+	}
+}   
+
+function studentTotalMarks(int maths, int physics, int chemistry, *Grades grades) {
+	int totalMarks = maths + physics + chemistry;
+	foreach [string, int?] [subject, marks] in grades.entries() {
+    	totalMarks = totalMarks + <int>marks;
+	}
+}
+```
+
+A named argument in a function call can correspond to an included record parameter in two different ways.
```

Comment:
```suggestion
A named argument in a function call can correspond to the fields of an included record parameter in two different ways.
```

If we don't say field, IMO there are three ways including passing the record value itself as a named arg.

Reply:
Included in https://github.com/praneesha/ballerina-dev-website/pull/10